src/vm/i386/asmhelpers.S

```diff
@@ -815,6 +815,13 @@ PATCH_LABEL ExternalMethodFixupPatchLabel
 NESTED_END ExternalMethodFixupStub, _TEXT
 
 #ifdef FEATURE_READYTORUN
+NESTED_ENTRY DynamicHelperArgsStub, _TEXT, NoHandler
+    CHECK_STACK_ALIGNMENT
+    call  eax
+    add   esp, 12
+    ret
+NESTED_END DynamicHelperArgsStub, _TEXT
```

Comment:
As I understand, JIT already treats the dynamic helpers as CDECL. The problem comes from something like argument shuffling stub created by ``DynamicHelpers::CreateHelperWithTwoArgs``.